### ISSUE TITLE
Error modal not properly covering all payment controls

### ISSUE THREAD

**Issue by kjozwiak** [brave/browser-laptop#14062]
### Description

When you receive the corrupted seed modal, you'll notice that the `Show only included sites` isn't fully covered and can still be used. The modal should be covering the entire area and not give the user the ability to use any controls until the seed is restored.

### Steps to Reproduce

* Run through the STR included in #14001

**Actual result:**

Scenario 1 :
<img width="841" alt="screen shot 2018-05-08 at 4 15 47 pm" src="https://user-images.githubusercontent.com/2602313/39780688-1908b872-52db-11e8-8196-5db3e5e64b65.png">
Scenario 2: 
![image](https://user-images.githubusercontent.com/17010094/40725671-0569377e-63f2-11e8-9a83-ff1b2d174cb2.png)

**Expected result:**

`Show only included sites` shouldn't be visible when the corrupted seen modal is visible.

**Reproduces how often:**

100% reproducible using the above STR.

### Brave Version

**about:brave info:**

 ```
Brave: 0.22.703 
V8: 6.6.346.26 
rev: 903b8d0d3703b5fdc5a719294403e7460d9b806d 
Muon: 6.0.8 
OS Release: 17.4.0 
Update Channel: Beta 
OS Architecture: x64 
OS Platform: macOS 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 66.0.3359.13
```

**Reproducible on current live release:**

N/A, the corrupted seed modal regressed in `0.22.669`.

**Comment by srirambv** [comment]
Happens on this as well 
![image](https://user-images.githubusercontent.com/17010094/39782942-90013408-52e1-11e8-8f38-44d0891d452b.png)

**Comment by kjozwiak** [comment]
What's strange is that now it's not even getting close to covering `Show only included sites` as it did before despite using `0.22.703` which was used in the original issue...

<img width="850" alt="screen shot 2018-05-08 at 5 26 40 pm" src="https://user-images.githubusercontent.com/2602313/39784042-11b90f86-52e5-11e8-9023-6a3253315a7b.png">

**Comment by NejcZdovc** [comment]
@kjozwiak `Show only included sites` should not be covered with this dialog

**Comment by NejcZdovc** [comment]
Pushing this back into 0.23 as fix made is #14067 will not cover all the cases, so we will need to create more general solution
